mysql/infoschema.go

```diff
@@ -375,6 +380,42 @@ func getForeignKeys(conv *internal.Conv, db *sql.DB, table schemaAndName) (forei
 	return foreignKeys, nil
 }
 
+// getIndexes return list all the indexes.
+func getIndexes(conv *internal.Conv, db *sql.DB, table schemaAndName) ([]schema.Index, error) {
+	q := `SELECT DISTINCT INDEX_NAME,COLUMN_NAME,SEQ_IN_INDEX,COLLATION,NON_UNIQUE
+		FROM INFORMATION_SCHEMA.STATISTICS 
+		WHERE TABLE_SCHEMA = ?
+			AND TABLE_NAME = ?
+			AND INDEX_NAME != 'PRIMARY' 
+		ORDER BY INDEX_NAME, SEQ_IN_INDEX;`
+	rows, err := db.Query(q, table.schema, table.name)
+	if err != nil {
+		return nil, err
+	}
+	defer rows.Close()
+	var name, column, sequence, collation, nonUnique string
+	indexes := make(map[string]schema.Index)
+	var indexNames []string
+	var indexArr []schema.Index
```

Comment:
Is Arr short for array?

Suggest: indexArr --> indexes
since this isn't an array (it's a slice).
And maybe using indexMap for the current indexes variable?

Reply:
updated variable names